src/accelerate/hooks.py

```diff
@@ -371,6 +411,11 @@ def attach_align_device_hook_on_blocks(
             Whether or not to include the associated module's buffers when offloading.
         module_name (`str`, *optional*, defaults to `""`):
             The name of the module.
+        load_all_weights_classes (`List[str]`, *optional*):
```

Comment:
```suggestion
        preloaded_modules (`List[str]`, *optional*):
```

What about something such as `preloaded_modules`?

Reply:
Maybe `preloaded_module_classes` to be similar to `no_split_module_classes`? I like `preloaded` a lot, thanks for the suggestion!

Reply:
Works with me!